lib/experimental/Navigation/Sidebar/Searchbar/index.tsx

```diff
@@ -19,7 +19,7 @@ export function SearchBar({
     <button
       onClick={onClick}
       className={cn(
-        "flex w-full cursor-pointer items-center justify-between rounded border border-solid border-f1-border bg-f1-background p-1.5 text-f1-foreground-secondary transition-colors hover:border-f1-border-hover",
+        "mb-4 flex w-full cursor-pointer items-center justify-between rounded border border-solid border-f1-border/70 bg-f1-background/80 p-1.5 text-f1-foreground-secondary transition-colors hover:border-f1-border-hover",
```

Comment:
is there a way to use a gap or padding of the container, not a margin?

Reply:
I tried it but couldn't find a way, as the spacing between elements isn't consistent.